frontend/public/components/monitoring/_monitoring.scss

```diff
@@ -1,5 +1,10 @@
 $tooltip-background-color: #151515;
 
+.co-resource-item.co-resource-item--monitoring-alert {
+  display: inline-flex;
```

Comment:
Does the `inline` prop on `ResourceLink` work OK instead?

Reply:
Thanks. I added the `inline` prop (although we still need `.co-resource-item--monitoring-alert` to set the desired margin).